features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/model/event/TimelineItemEventContentProvider.kt

```diff
@@ -61,6 +61,8 @@ class TimelineItemTextBasedContentProvider : PreviewParameterProvider<TimelineIt
         aTimelineItemNoticeContent().copy(formattedBody = buildSpanned("Notice")),
         aTimelineItemTextContent(),
         aTimelineItemTextContent().copy(formattedBody = buildSpanned("Text")),
+        // Emojis should render larger than normal text
```

Comment:
For some reason this doesn't seem to work in the preview. I'd guess it's because the preview doesn't have the emoji library initialised (https://github.com/element-hq/element-x-android/pull/3295/files#diff-78bca535fd89d019ca42a19f0ee787cd9e0ad3d8072433a4a7f6c18c8b6b19b5R42)? Not sure how to make that work

Reply:
I don't think there's an easy way to preview these, we could add some check along with `LocalInspectionMode.current` maybe, but that seems like quite the hack.